data/xapleaux/xapleaux missions.txt

```diff
@@ -0,0 +1,182 @@
+			`	It launches into a long string of chirps, whistles, clicks, and trills, accompanied by an intensely complex series of gestures, which it repeats several times. Suddenly, as the sun rising on an airless moon, it dawns on you that whatever language these aliens use, it is clearly at least half some kind of gesture-based sign-language.`
+			`	Experimentally you ask, "Is this how you talk?" in Remnant sign. The alien does not reply in any words or gestures that you understand, but the effect is nevertheless very dramatic. The alien verily shrieks, its entire body quivering in some emotion barely constrained.`
+
+			`	Excitement? Anger? Fascination? Concern?`
+			`	You can't be sure, but it doesn't seem negative as another rushes over from a nearby building and it gestures at you until you repeat the question in sign again. Slowly, and carefully, the two aliens mimic your actions with their middle arms.`
+			`	In an instant you realize that you are well out of your personal depth, but now you know exactly where you need to go next. At length you indicate the signs for 'I' and 'you', and manage to gesture-indicate your way through a sentence to tell them, "I will go, and come back again."`
+			`	Slowly, tentatively, painstakingly, the alien nods and carefully signs back, "You, go and come back again."`
+			`	So great is the tension as you board your ship again, you suspect that had you moved any slower they'd have carried you in there themselves.`
+				flee
+#This is accompanied by a mission that triggers on the same conditions as the branch to direct you to where you go. This is used so that multiple different sets of conditions can be used to resolve this in various different ways when alternative first contact options become available.
+	on defer
+		"Met X Already" ++
+
+
+#The below are commented out until more than just the landing mission are ready.
+
+#mission "Discovered Xapleaux Space"
+#	landing
+#	invisible
+#	source
+#		government "Xapleaux"
+#	on offer
+#		event "label the brilliance"
+#		fail
+
+#event "label the brilliance"
+#	galaxy "label brilliance"
+#		sprite "label/brilliance"
+
+
+
+mission "Xapleaux: First Contact In Progress"
+	landing
+	source
+		government "Xapleaux"
+		not attributes "uninhabited"
+	to offer
+		has "Xapleaux: First Contact: declined"
+
+	on offer
+		conversation
+			`	Upon disembarking your ship you make sure you have your pink crystal token to hand, and your ship is immediately set upon by the various aliens doing refuelling and maintenance with the rapid-paced manner of communication that seems to be typical for them.`
+			`	When they are finished, one of them approaches you, points at your ship, and gives two sharp whistles and a nod. Time to go.`
+
+			`	We know we need the help of the Remnant if we're going to make better contact with them.`

```

Comment:
Any reason for the use of 'we' in this sentence instead of just 'you'?

Reply:
Probably a leftover copy-paste article from a prior summary.
```suggestion
			`	You know you will need the help of the Remnant if you're going to make better contact with them.`

```